common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -213,6 +213,37 @@ def test_auto_enroll_csv_a11y(self):
         self.auto_enroll_section.a11y_audit.check_for_accessibility_errors()
 
 
+class BatchBetaTestersTest(BaseInstructorDashboardTest):
```

Comment:
Not sure if it should be covered in this scope but there are other scenarios related to batch beta testers related to you change. For example test for when there are active and inactive users together to be added as beta testers.